TACOS-facade/src/main/java/cz/fi/muni/TACOS/dto/AttributeCategoryCreateDTO.java

```diff
@@ -12,8 +11,6 @@
 
 	private String name;
 
-	private BigDecimal minimalPrice;
-
 	private Set<Long> attributes = new HashSet<>();
```

Comment:
Because you use in otherClass name for attributes something like this 'attributeIds', I think that we should use this in all these attributes in all class.